src/Controllers/mySQLQuery.java

```diff
@@ -30,11 +30,26 @@ public void initDB() {
      * @return - ResultSet of all products with qty_on_hand > 0.
      */
     public ResultSet getProdInInventory() throws SQLException {
-        String query = "select * from product where qty_on_hand > 0";
+        String query = "select * from product";
```

Comment:
I had to make the inventory function display all items regardless of whether we currently have any in stock otherwise there's no real way for the user to find anything with qty 0 in order to update it.

Reply:
that totally makes sense! nice catch